src/lib/utils/firebase-loader.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @fileoverview Dynamic loading code for Firebase and libraries.
+ */
+
+/**
+ * @type {!Object<!Promise>}
+ */
+const libs = {};
+
+const firebasePrefix = '//www.gstatic.com/firebasejs/6.6.1';
+
+/**
+ * @param {string} library to preload
+ * @return {!Promise} for library
+ */
+function internalPreload(library) {
```

Comment:
is this preloading or is it loading?

Reply:
I've removed references to 'preload' as that clearly has a strict meaning on the web, one which we're not using (i.e., `modulepreload` and friends).